apps/antalmanac/src/components/RightPane/SectionTable/GradesPopup.tsx

```diff
@@ -69,12 +69,12 @@ function GradesPopup(props: GradesPopupProps) {
                   instructor ? ` — ${instructor}` : ''
               } | Average GPA: ${gradeData.courseGrades.averageGPA.toFixed(2)}`
             : 'Grades are not available for this class.';
-    }, [deptCode, instructor, gradeData]);
+    }, [gradeData, deptCode, courseNumber, instructor]);
 
-    const gpaString = useMemo(
-        () => (gradeData ? `Average GPA: ${gradeData.courseGrades.averageGPA.toFixed(2)}` : ''),
-        [gradeData]
-    );
+    // const gpaString = useMemo(
```

Comment:
why does this need to be commented out?

Reply:
Wasn't being used, so I took some liberties since I was in the file 😅 . Can revert if pref.

Reply:
eh it's whatever. This tech debt isn't very high-interest.